frontend/src/components/static_pages/about.css

```diff
@@ -1,154 +1,146 @@
+.container {
+  padding: 0px 50px 25px 50px;
+}
 #aboutPageTextRow {
-  padding-left: 133px;
-  padding-right: 133px;
+  margin: 0px;
   display: block;
 }
-
 .sub-heading {
   font-size: 24px;
+  padding: 0px;
+  width: 100%;
 }
 
 .about-people-row {
-  margin-bottom: 20px;
+  margin: 40px 0px;
 }
+
 .images {
-  margin-bottom: 60px;
-  margin-left: 120px;
-  margin-right: 40px;
+  display: flex;
+  align-items: center;
 }
 
 .images img {
-  width: 330px;
+  width: 85%;
   border: 5px solid #d3d3d3;
 }
 
-#kim-image {
-  margin-top: 90px;
-  margin-bottom: 30px;
-  height: 350px;
-}
-
-#pierre-image {
-  margin-top: 30px;
-  margin-bottom: 30px;
-  height: 350px;
-}
-
-#shirley-image {
-  margin-top: 30px;
-  height: 355px;
-}
-
 .about-heading {
-  margin-left: 120px;
-  margin-bottom: 10px;
-  padding: 15px 0px;
+  margin: auto;
+  font-size: 48px;
   font-weight: bold;
+  width: 100%;
+  text-align: center;
 }
 
 .people-heading {
-  margin-top: 45px;
-  font-size: 48px;
+  font-size: 40px;
   font-weight: bold;
   display: inline-block;
+  width: 100%;
 }
 
-/* tablet */
+@media (min-width: 2000px) {
+  .container {
+    max-width: 2000px;
+  }
 
-@media all and (min-width: 768px) and (max-width: 1023px) {
-  #aboutPageTextRow {
-    padding-left: 20px;
-    padding-right: 20px;
+  p {
+    font-size: 2rem;
```

Comment:
please use `px` instead of `rem`

Reply:
rem's have been removed and switched them to px... thank you! :)